### ISSUE TITLE
Other locales

### ISSUE THREAD

**Issue by mpavlenko** [Da-Teach/RaidStaticData#2]
Hi, @Da-Teach  thank you a lot for this static data. Could you add more localizations for static data, or share plz a way how to get this data (Previously I extracted that from windows registry - it was base64 encoded gzip, but after one of updates I was unable to do that)

**Comment by Happy-Minion** [comment]
@Da-Teach amazing work.
I would love to learn how you extracted the static data getting the damage modifiers is such a great help and with all the new champions coming out constantly it would be great to be able to extract this data with every patch that introduces a new champ

**Comment by Da-Teach** [comment]
Basically the way I get this data is by injecting a DLL into the game which reads all the ingame data and serializes it to json (all using ingame functions actually). It's all part of a custom bot that I wrote for private use (although tempting to release it). The bot is a, uhm, bit more advanced than the average clicker ;)

As for other locales, seeing as the game apparently loads only one language, it would mean running the code for each locale that you want. Not impossible, but for me not practical. Currently don't have time otherwise I'd write a small static data extractor.

**Comment by Happy-Minion** [comment]
OK, thanks for the info. Guess we will rely on you then for the data every patch.
If you ever gonna decide to stop this, please be so kind and share the DLL with some basic description in the comments with us (or with me).
Big thanks for putting in all the time into coding this.

**Comment by Doorjam** [comment]
Thanks @Da-Teach! I just saw this right now, would be awesome if you could share the method when you do decide to stop. 
You've helped me so much with just this data and I'm sure countless others!

**Comment by omgmarc** [comment]
> It's all part of a custom bot that I wrote for private use (although tempting to release it). The bot is a, uhm, bit more advanced than the average clicker ;)

Would love to poke your brain about this subject or collaborate on it. Now that it seems Plarium is indeed OK with reading the game client's memory, I am slowly rewriting my auto clicker script into what I originally wanted to create.

**Comment by Da-Teach** [comment]
While they are fine with memory reading (for now), I'm 99.999% sure that they will not approve/condone/accept a full blown bot that turns Raid into "Idle Raid".

**Comment by omgmarc** [comment]
> While they are fine with memory reading (for now), I'm 99.999% sure that they will not approve/condone/accept a full blown bot that turns Raid into "Idle Raid".

Right, that's definitely not the objective, though...at least not with mine. And AFAIK, they are aware of OMGclick and haven't had any issue with it (i don't see why they would, considering it, in a way, does generate more revenue for them by means of energy/gem refreshes). There are just some aspects in which I would prefer not to have to rely on the "visual" detection method I've been using thus far...specifically the rewards. It will also allow me to implement some long-standing feature requests regarding session data (e.g. champ damage, run times, etc), which would be quite the undertaking to do so using pattern detection.

That said, I do understand if you'd rather not aid me in this endeavor. Figured it doesn't hurt to ask. :)

**Comment by grBIN** [comment]
> As for other locales, seeing as the game apparently loads only one language, it would mean running the code for each locale that you want. Not impossible, but for me not practical. Currently don't have time otherwise I'd write a small static data extractor.

Hi, I understand that every time downloading all languages and data for them is a very long time and not practical, but maybe you could get 1-2 additional most used ones or on request?
Accidentally stumbled upon this repository, and I would very much like to receive data, including in Russian, thanks in advance =)
